crates/goose-mcp/src/google_drive/mod.rs

```diff
@@ -45,6 +45,13 @@ enum FileOperation {
     Create { name: String },
     Update { file_id: String },
 }
+#[derive(PartialEq)]
+
```

Comment:
nit: remove the newline

Reply:
Eeew gross. How did that even happen? I'm telling you, I don't trust these tools sometimes! Fixed.